lib/src/super_extensions_base.dart

```diff
@@ -83,3 +83,13 @@ extension StringExtension on String {
           .map((str) => str.capitaliseFirstChar())
           .join(' ');
 }
+
+extension NumberExtension on num {
+  /// creates a Empty [SizedBox] with height as the given number
+  /// 
+  SizedBox get pH => SizedBox(height: toDouble());
```

Comment:
I would suggest, having a name as `pH` can be a bit confusing. For example, if I don't open the extension file, I won't understand that pH creates a SizedBox with height. Maybe having name like `hSizedBox` or something else that makes more sense?

Reply:
Also, please update the `Readme.md` file too to include the new extension methods.

Reply:
Yes, That makes sense. I'll update it.